tests/model/test_model.py

```diff
@@ -963,10 +995,19 @@ def test_get_all_users(self, mocker, initial_data, user_list, user_dict, user_id
         assert model.user_dict == user_dict
         assert model.users == user_list
 
-    @pytest.mark.parametrize("muted", powerset([1, 2, 99, 1000]))
-    def test__subscribe_to_streams(self, initial_data, muted, model):
+    @pytest.mark.parametrize(
+        "muted, desktop_notifs_enabled",
```

Comment:
Let's be consistent with variable naming and use `*_notifications_*` everywhere.